.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on: [push, pull_request, workflow_dispatch]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.3
+     
+    - name: CI - Verifications and Tests
+      run: |
+        make toolchain 
+        echo "/usr/local/kubebuilder/bin" >> $GITHUB_PATH
+        make ci
+
+    - name: Upload code coverage
+      uses: actions/upload-artifact@v2
+      with:
+        name: coverage
```

Comment:
Is name required here

Reply:
name isn't required but I think this makes it more readable in the Github actions summary view where it just says "Run actions/upload-artifact@v2" which is pretty unclear as to what it's doing :)

Reply:
Agree that for this step (and other parameterized actions), adding a name can increase clarity.